src/main/java/bio/terra/service/filedata/DrsService.java

```diff
@@ -75,6 +77,11 @@ public class DrsService {
 
   private final Logger logger = LoggerFactory.getLogger(DrsService.class);
 
+  private static final String ACCESS_ID_PREFIX_GCP = "gcp-";
+  private static final String ACCESS_ID_PREFIX_AZURE = "azure-";
```

Comment:
I think this was `az-` originally.  Is it ok that it would change?  Should we leave it as `az-`?